main/busybox/0011-sysklogd-add-Z-option-to-adjust-message-timezones.patch

```diff
@@ -79,7 +79,7 @@ index 4265f4f90..eca955891 100644
 -		now = 0;
 -		timestamp = msg;
 +		if (G.adjustTimezone && strptime(msg, "%b %e %T", &nowtm)) {
-+			now = mktime(&nowtm) - timezone;
++			now = timegm(&nowtm) - timezone;
```

Comment:
While at it we might as well check the return value of timegm, any thoughts on this?